src/main/resources/base/store

```diff
@@ -0,0 +1 @@
+{"token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjVjZGU1MTk5ODdlY2U1MDAxMDBlMDMxMiIsImlhdCI6MTU3MTk3MTU1MywiZXhwIjoxNTcyMDU3OTUzLCJpc3MiOiJodHRwczovL2RlZXBtYXJrZXQuY3MucGR4LmVkdSJ9.c4w5_lMxDon282aIT89mno6fBRStdljCvfa35I6pVg0"}
```

Comment:
probably shouldn't be in source control

Reply:
Yikes -- I could have sworn I double checked this to make sure it wasn't in here. Good call and too bad it'll be in revision history forever now : /

Reply:
https://help.github.com/en/github/authenticating-to-github/removing-sensitive-data-from-a-repository this might help

Reply:
Gotcha. I removed and invalidated the token.